executor.go

```diff
@@ -316,8 +317,12 @@ func (e *Executor) taskEventLoop() {
 		case Kill:
 			e.shutDown(taskInfo, cmd)
 			message := "Task killed due to receiving an event from Mesos agent"
+			taskID := "MISSING"
```

Comment:
I'd rather have this: `taskID := mesos.TaskID{Value:"MISSING"}`

Reply:
@AlfredBroda fixed.